src/objects/application.ts

```diff
@@ -26,6 +26,7 @@ function createApplication(stage: HTMLDivElement): void {
   application = new Application({
     background: '#1099bb',
     resizeTo: stage,
+    antialias: true,
```

Comment:
Antialiasing will help, but you'll def want the `resolution: window.devicePixelRatio || 2,` so that it uses the proper screen px density